### ISSUE TITLE
Fork doesn't show commits if repo contains a file named HEAD

### ISSUE THREAD

**Issue by ericchan1341** [ForkIssues/TrackerWin#284]
![image](https://user-images.githubusercontent.com/50191093/58168764-a4631500-7cc1-11e9-8474-1df2dff0a787.png)
As you can see there is no branch UI whatsoever. I can switch/create branch but nothing will display. Other repo are able to display normally. All the repo are in github and they are created in the same fashion.

**Comment by DanPristupov** [comment]
Hard to say. Check `%localappdata%\fork\logs\fork.log`

**Comment by ericchan1341** [comment]
```I 2019-05-22 19:03:47.648 Log target: AppData log file
I 2019-05-22 19:03:47.772 Last update check: 21/5/2019 8:51:25 PM. Next one: 22/5/2019 8:51:25 PM
I 2019-05-22 19:03:47.932 Restoring 2 sessions.
I 2019-05-22 19:03:48.415 RefreshRepositoryData
I 2019-05-22 19:03:48.415 Main: 918, Window: 1485, WindowLoaded: 283, UiReady: 488, Total: 2256
D 2019-05-22 19:03:48.422 Esports RefreshRepositoryDataGitCommand All
D 2019-05-22 19:03:48.474 Loading Esports settings.
W 2019-05-22 19:03:48.723 Git request caused error 'log --branches --remotes --tags HEAD --max-count=25000 --date-order --pretty=format:%H
%T
%P
%an
%ae
%at
%s':
fatal: ambiguous argument 'HEAD': both revision and filename
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

I 2019-05-22 19:03:48.723 Refresh 'Esports' data. Updated.
I 2019-05-22 19:03:48.804 RefreshRepositoryStatus
I 2019-05-22 19:03:48.962 Refresh 'Esports' status. Updated 0 files
I 2019-05-22 19:03:48.975 RefreshAheadBehindInfo
I 2019-05-22 19:04:47.660 Automatically fetching 'CurseOfTheRoyalFortune'
I 2019-05-22 19:04:48.758 Automatically fetching 'Esports'
I 2019-05-22 19:04:51.994 RefreshRepositoryData
D 2019-05-22 19:04:51.994 Esports RefreshRepositoryDataGitCommand -500
D 2019-05-22 19:04:52.049 Detected changed reference
W 2019-05-22 19:04:52.098 Git request caused error 'log --branches --remotes --tags HEAD --max-count=25000 --date-order --pretty=format:%H
%T
%P
%an
%ae
%at
%s':
fatal: ambiguous argument 'HEAD': both revision and filename
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

I 2019-05-22 19:04:52.098 Refresh 'Esports' data. Updated.
I 2019-05-22 19:04:52.114 RefreshAheadBehindInfo
I 2019-05-22 19:08:45.265 RefreshRepositoryData
D 2019-05-22 19:08:45.270 Saving settings.
D 2019-05-22 19:08:45.270 Esports RefreshRepositoryDataGitCommand All
I 2019-05-22 19:08:45.284 RefreshRepositoryData
D 2019-05-22 19:08:45.305 CurseOfTheRoyalFortune RefreshRepositoryDataGitCommand All
D 2019-05-22 19:08:45.710 Loading CurseOfTheRoyalFortune settings.
D 2019-05-22 19:08:46.232 Esports Repository data are equal. No Changes
I 2019-05-22 19:08:46.240 RefreshRepositoryStatus
I 2019-05-22 19:08:46.701 Refresh 'CurseOfTheRoyalFortune' data. Updated.
I 2019-05-22 19:08:46.806 RefreshRepositoryStatus
I 2019-05-22 19:08:46.896 Refresh 'Esports' status. Updated 0 files
I 2019-05-22 19:08:46.896 RefreshAheadBehindInfo
I 2019-05-22 19:08:47.375 Refresh 'CurseOfTheRoyalFortune' status. Updated 12 files
I 2019-05-22 19:08:47.401 RefreshAheadBehindInfo
I 2019-05-22 19:08:50.209 RefreshRepositoryStatus
I 2019-05-22 19:08:50.408 Refresh 'CurseOfTheRoyalFortune' status. Updated 12 files
D 2019-05-22 19:08:55.142 Saving settings.
I 2019-05-22 19:08:55.142 RefreshRepositoryData
D 2019-05-22 19:08:55.142 Esports RefreshRepositoryDataGitCommand All
D 2019-05-22 19:08:55.449 Esports Repository data are equal. No Changes
I 2019-05-22 19:08:55.455 RefreshRepositoryStatus
I 2019-05-22 19:08:55.604 Refresh 'Esports' status. Updated 0 files
I 2019-05-22 19:08:55.604 RefreshAheadBehindInfo
I 2019-05-22 19:09:00.220 RefreshRepositoryData
D 2019-05-22 19:09:00.220 Esports RefreshRepositoryDataGitCommand -115
D 2019-05-22 19:09:00.302 Detected changed active branch
W 2019-05-22 19:09:00.354 Git request caused error 'log --branches --remotes --tags HEAD --max-count=25000 --date-order --pretty=format:%H
%T
%P
%an
%ae
%at
%s':
fatal: ambiguous argument 'HEAD': both revision and filename
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

I 2019-05-22 19:09:00.354 Refresh 'Esports' data. Updated.
I 2019-05-22 19:09:00.367 RefreshRepositoryStatus
I 2019-05-22 19:09:00.702 Refresh 'Esports' status. Updated 0 files
I 2019-05-22 19:09:00.707 RefreshAheadBehindInfo
I 2019-05-22 19:20:38.026 RefreshRepositoryData
D 2019-05-22 19:20:38.026 Esports RefreshRepositoryDataGitCommand All
D 2019-05-22 19:20:38.097 Saving settings.
D 2019-05-22 19:20:38.124 Saving settings.
D 2019-05-22 19:20:38.139 Saving settings.
I 2019-05-22 19:20:38.149 Stop ipc server Fork_2816_Pipe_AskPass```

**Comment by DanPristupov** [comment]
> ambiguous argument 'HEAD': both revision and filename

It looks like you have a file `HEAD` in your repository which conflicts with the `HEAD` reference. Do you have such file?

I need to add `--` as a suffix to the git request to fix this. Please wait for the next update or remove the file.

**Comment by DanPristupov** [comment]
Fixed in 1.34
